buildtest/menu/build.py

```diff
@@ -111,6 +149,35 @@ def func_build_subcmd(args):
     # Discover list of one or more config files based on path provided
     config_files = discover_configs(args.config)
 
+    # if no files discovered let's stop now
+    if not config_files:
+        msg = "There are no config files to process."
+        logger.error(msg)
```

Comment:
What is the reason for wanting the message to go to the logger.error _and_ system.exit?

Reply:
Good point, i don't have any particular reasoning. I guess it doesn't make sense to have same error message if printed to screen. I guess it would be only useful if an exception message is captured before exiting for detailed error message. That's what i am accustomed to, just ``tail``  the last few lines to get the error message.

Reply:
I was thinking it would be cleaner to do something like:

```python
logger.error("There was an error")  # <--- error message goes to log
sys.exit(1)                                       # <--- exit with error code
```

Reply:
I will remove this line and fixed locally.